bookmarklet-js/mediaQueryBookmarklet.js

```diff
@@ -180,7 +180,7 @@ window.mqb = {
     mqb.css = document.createElement( 'link' );
     mqb.css.type = "text/css";
     mqb.css.rel = "stylesheet";
-    mqb.css.href = "http://sparkbox.github.com/mediaQueryBookmarklet/stylesheets/mediaQuery.css";
+    mqb.css.href = "https://sparkbox.github.io/mediaQueryBookmarklet/stylesheets/mediaQuery.css";
```

Comment:
Should the styles sheets live in the `gh-pages` branch, since that it is what it’s linking to or in `master`?